src/components/Breadcrumb/BreadcrumbItem.tsx

```diff
@@ -1,24 +1,29 @@
+import { Link, LinkElement } from "./Breadcrumb"
 import styles from "./Breadcrumb.module.css"
 
-export interface LinkProps {
-  href: string
-  label: string
-  isCurrent?: boolean
+export interface BreadcrumbItemProps {
+  isCurrent: boolean
+  link: Link
+  linkElement?: LinkElement
 }
 
-export const BreadcrumbItem = ({ href, isCurrent, label }: LinkProps) => {
+export const BreadcrumbItem = ({
+  isCurrent,
+  link,
+  linkElement,
+}: BreadcrumbItemProps) => {
   const ariaCurrent = isCurrent ? "page" : undefined
 
   return (
-    <li className={styles.BreadcrumbItem}>
-      <a href={href} className={styles.link} aria-current={ariaCurrent}>
-        {label}
-      </a>
-      {!isCurrent && (
-        <span aria-hidden="true" className={styles.Separator}>
-          /
-        </span>
+    <li className={styles.list}>
```

Comment:
⛏️ It feels odd to label a list item as a "list"...
```suggestion
    <li className={styles.listItem}>
```